### PR TITLE
Client can now create, update, and delete from events, event_registration, campgrounds, and campground_reservations

### PR DESCRIPTION
**Pull request by vanessaspear**
# Description
Added create, update, and delete functions and implemented create_resource sql helper function into events, event_registration, campgrounds, and campground_reservations 

# Steps to test
- Fetch the vs-method-mapper branch and switch to it
- Start/restart debugger
- Send a GET request to campgrounds and copy a single dictionary from the response list
- Switch to the POST function, and paste the copied dictionary into the Body field.  Remove the "id" line. 
- Send a POST request to add a campground (e.g. http://localhost:8088/campgrounds)
- Verify that a 201 status code and the new dictionary are received
- Copy the dictionary again, and switch to the PUT function. 
- Paste the dictionary into the Body field.  Change any value (except the "id").
- Send a PUT request to update the campground (e.g. http://localhost:8088/campgrounds/1)
- Verify that a 204 status code was received and switch back to the GET function.
- Request all campgrounds, and verify the dictionary was updated
- Send a DELETE request to delete the dictionary (e.g. http://localhost:8088/campgrounds/1)
- Request all campgrounds, and verify the dictionary was removed from the list

Repeat process for the following GET resources:
http://localhost:8088/events
http://localhost:8088/event_registration
http://localhost:8088/camping_reservations

### CODE REVIEW

File: views/event_registration_requests.py [hunk 1]
```diff
@@ -49,4 +50,67 @@ def get_single_registration(id):
...
+        new_event_registration (dict): Event_registration dictionary with updated values
+    """
+    with sqlite3.connect("./national_park.sqlite3") as conn:
+        db_cursor = conn.cursor()
+
+        db_cursor.execute("""
+        UPDATE event_registration
+            SET
+                event_id = ?,
+                user_id = ?,
```
Comment (mvdutta) [inline views/event_registration_requests.py]: need to remove comma on line 104

**Review by mvdutta — CHANGES_REQUESTED** [review]
Everything is working except for PUT request on event_registrations. Just need to remove a comma on 104

File: views/event_registration_requests.py (hunk 1 above)
Comment (vanessaspear) [reply views/event_registration_requests.py]: Good catch, I just updated and pushed my changes.

**Review by mvdutta — APPROVED** [review]
Awesome! Loved the extremely clear test instructions!

**Pull request merged**